### DISCUSSION TITLE
help, how to fix this error

### DISCUSSION THREAD

**Discussion by willyw2k** [alpinejs/alpine#494] [4. General]
im trying validate html using w3c validator but i got some error

![Screenshot_2020-05-16-02-03-43-32](https://user-images.githubusercontent.com/11731303/82087406-39082380-971a-11ea-925d-d091f203ef9e.jpg)
how to fix this?

**Comment by ryangjchandler** [comment]
Hi @willyw2k, unfortunately since Alpine uses a custom attribute schema, it does not produce valid HTML according to the W3C spec so there isn't any way to fix this currently.

Some discussions on the problem can be found at #450.
